glide.lock

```diff
@@ -14,18 +14,27 @@ imports:
   - quantile
 - name: github.com/blang/semver
   version: b38d23b8782a487059e8fc8773e9a5b228a77cb6
+- name: github.com/BurntSushi/toml
+  version: 3012a1dbe2e4bd1391d42b32f0577cb7bbc7f005
 - name: github.com/certifi/gocertifi
   version: ee1a9a0726d2ae45f54118cac878c990d4016ded
+- name: github.com/containerd/continuity
+  version: 75bee3e2ccb6402e3a986ab8bd3b17003fc0fdec
+  subpackages:
+  - pathdriver
 - name: github.com/containers/image
-  version: 4bc6d24282b115f8b61a6d08470ed42ac7c91392
-  repo: https://github.com/openshift/containers-image.git
+  version: abf32c4ea589cb8e96bdca5a478dba68f11980e5
```

Comment:
This is v2.0.0; please use at least v3.0.0.

Reply:
@mtrmac We don't specify a containers/image version in `glide.yaml`

This version in `glide.lock` comes from the versions we do specify, which in this case is machine-config-operator.

And if you look there:  https://github.com/openshift/machine-config-operator/blob/master/Gopkg.toml#L168-L170 and https://github.com/openshift/machine-config-operator/blob/release-4.2/Gopkg.toml

It specifies v2.0.0.0:

```
[[constraint]]
  name = "github.com/containers/image"
  version = "v2.0.0"
```

You want use to use v3.0.0.0, then update machine-config-operator accordingly and we can refresh our dependency once that is done.

I do not want to get into the business of specifying containers/image if we are not coding to it specifically.  And you'll need to give me a good, precise reason why we have to run at a different verison of that vs. machine-config-operator.

Reply:
@mtrmac and I talked offline in slack ... we've agreed to live with the current version of containers/image in this PR.

I'll queue up a separate discussion with @adambkaplan and @bparees the pros and cons for possible explicit citation of containers/image in the OCM repo as part of a subsequent change.